app/src/app/app.component.html

```diff
@@ -12,10 +12,27 @@
 See the License for the specific language governing permissions and
 limitations under the License. -->
 
-<app-navbar></app-navbar>
+<div class="nav-bar">
+  <button id="extend-button" mat-icon-button (click)="snav.toggle()">
+    <mat-icon>menu</mat-icon>
+  </button>
+  <img
+    (click)="setRoute('home')"
+    src="../../../assets/img/specmath_logo.svg"
```

Comment:
Can you link directly to /assets/img/specmath_logo.svg instead of using a relative path?

Reply:
Updated